README.md

```diff
@@ -328,6 +328,8 @@ For Puppet 4.2.2+ (and 3.8.2) we've lessened the possibility of the issue occurr
 
 If you are affected by this issue, a known workaround is to download the `.tar.gz` from the forge and use `puppet module install` using the file rather than directly from the forge.
 
+- Windows 2003 is not supported.  If installing the module to the default `production` environment on a Puppet master, pluginsync will attempt to install on Windows 2003 nodes within the `production` environment.  This will likely cause pluginsync failures, where the LFN issue is even more problematic (due to a longer user home directory). In this situation, [Puppet environments](https://docs.puppetlabs.com/puppet/latest/reference/environments.html) may be used to segregate nodes and control module installs.  Since modules may be configured independently per environment, this allows for the creation of an environment to isolate Windows 2003, and ensure the DSC module is not installed.
+
```

Comment:
This is good, but it doesn't really explain that the module will be synced down to Windows 2003 agents automatically and will cause issues. 

"If you have any Windows 2003 agents, this module will cause pluginsync to fail on those agents due to an issue with [LFN (Long File Name)](https://tickets.puppetlabs.com/browse/PUP-4866). As a workaround, consider putting your Windows 2003 agents in a separate environment isolated from your normal `production` environment, see [Puppet environments](https://docs.puppetlabs.com/puppet/latest/reference/environments.html)."